src/dotnet-install.ps1

```diff
@@ -96,6 +98,7 @@ param(
    [string]$FeedCredential,
    [string]$ProxyAddress,
    [switch]$ProxyUseDefaultCredentials,
+   [string]$ProxyBypassList="",
```

Comment:
Defaulting to empty string is not very safe because you can always pass explicit null when the parameter is not mandatory.

Reply:
Thanks @nohwnd, I have updated with the suggested changes.